pydicer/convert/convert_pt.py

```diff
@@ -0,0 +1,311 @@
+        ValueError: Raised when a modality or a unit (for the PT) is not
+                    handled.
+    Returns:
+        numpy.array: The numpy image, used to compute the bounding boxes
+    """
+    slices = [pdcm.read_file(str(dcm["path"])) for dcm in input_filepaths]
+    slices.sort(key=lambda x: float(x.ImagePositionPatient[2]))
+    if modality == "PT":
+        if hasattr(slices[0], "PatientWeight") and (slices[0].PatientWeight is None):
+            if hasattr(slices[0], "PatientsWeight"):
+                patient_weight = float(slices[0].PatientsWeight)
+            elif patient_weight_from_ct is not None:
+                patient_weight = patient_weight_from_ct
+            else:
+                # raise MissingWeightException(
+                #     'Cannot compute SUV the weight is missing')  # from Shuchao's code
+                patient_weight = 75.0  # From Shuchao's code (default)
+                warnings.warn(
+                    "Cannot find the weight of the patient, hence it "
+                    "is approximated to be 75.0 kg"
+                )
+        elif not hasattr(slices[0], "PatientWeight"):
+            if hasattr(slices[0], "PatientsWeight"):
+                patient_weight = float(slices[0].PatientsWeight)
+            elif patient_weight_from_ct is not None:
+                patient_weight = patient_weight_from_ct
+            else:
+                # raise MissingWeightException(
+                #     'Cannot compute SUV the weight is missing')  # from Shuchao's code
+                patient_weight = 75.0  # From Shuchao's code (default)
+                warnings.warn(
+                    "Cannot find the weight of the patient, hence it "
+                    "is approximated to be 75.0 kg"
+                )
+        else:
+            patient_weight = float(slices[0].PatientWeight)
+
+    # Check if all the slices come from the same serie
+    same_serie_uid = True
+    serie_uid = slices[0].SeriesInstanceUID
+    for s in slices:
+        same_serie_uid *= serie_uid == s.SeriesInstanceUID
+
+    if not same_serie_uid:
+        raise RuntimeError("A slice comes from another serie")
+
+    axial_positions = np.asarray([k.ImagePositionPatient[2] for k in slices])
+    # Compute redundant slice positions
+    ind2rm = [
+        ind
+        for ind in range(len(axial_positions))
+        if axial_positions[ind] == axial_positions[ind - 1]
+    ]
+    # Check if there is redundancy in slice positions and remove them
+    if len(ind2rm) > 0:
+        slices = [k for i, k in enumerate(slices) if i not in ind2rm]
+        axial_positions = np.asarray([k.ImagePositionPatient[2] for k in slices])
+
+    slice_spacing = (
+        slices[1].ImagePositionPatient[2] - slices[0].ImagePositionPatient[2]
+    )
+
+    pixel_spacing = np.asarray(
+        [
+            slices[0].PixelSpacing[0],
+            slices[0].PixelSpacing[1],
+            slice_spacing,
+        ]
+    )
+
+    if modality == "PT":

```

Comment:
Yeah again, if this function is only going to be used for converting PET, then this check and input parameter won't be needed in my opinion